spec/features/staff/users_can_create_a_budget_spec.rb

```diff
@@ -17,6 +17,24 @@
 
         expect(page).to have_content(I18n.t("form.budget.create.success"))
       end
+
+      scenario "budget creation is tracked with public_activity" do
+        fund_activity = create(:fund_activity, organisation: user.organisation)
+
+        PublicActivity.with_tracking do
+          visit organisation_path(user.organisation)
+          click_on(fund_activity.title)
+
+          click_on(I18n.t("page_content.budgets.button.create"))
+
+          fill_in_and_submit_budget_form
+
+          budget = Budget.last
+          auditable_event = PublicActivity::Activity.find_by(trackable_id: budget.id)
+          expect(auditable_event.key).to eq "budget.create"
+          expect(auditable_event.owner_id).to eq user.id
```

Comment:
Is it possible to `expect` to see that the event connects to the budget record? I can see it goes in as a param but I think the important thing is we can get access to it down the road.